webpack/react_app/redux/actions/job_invocations/index.js

```diff
@@ -0,0 +1,60 @@
+import { isNil } from 'lodash';
+
+import {
+  JOB_INVOCATIONS_GET_JOB_INVOCATIONS,
+  JOB_INVOCATIONS_POLLING_STARTED,
+} from '../../consts';
+
+import API from 'foremanReact/API';
+
+const defaultJobInvocationsPollingInterval = 10000;
+const jobInvocationsInterval = isNil(process.env.JOB_INVOCATIONS_POLLING) ?
+  defaultJobInvocationsPollingInterval :
+  process.env.JOB_INVOCATIONS_POLLING;
+
+const getJobInvocations = url => (dispatch) => {
+  const isDocumentVisible =
+    document.visibilityState === 'visible' ||
+    document.visibilityState === 'prerender';
+
+  if (isDocumentVisible) {
+    API.get(url)
+      .then(onGetJobInvocationsSuccess)
```

Comment:
This is the key part that changed because of https://github.com/theforeman/foreman/pull/5080